### DISCUSSION TITLE
Per-site proxy

### DISCUSSION THREAD

**Discussion by HamonTeacher** [hiddify/hiddify-app#1265] [Q&A]
Where is the split tunneling mode in the program? On Android I found it right away, but I didn't understand where this mode is in the Linux version. How to specify specific sites or applications for proxying?

**Comment by CapybarasLover** [comment]
same bro🏄‍♂️

**Comment by agrogers** [comment]
I couldn't find it in Windows either. But perhaps that is because you don't really need it. I am running Hiddify as a "Proxy" not a "System Proxy". Then in the Windows proxy settings I configure the Manual proxy and point it at Hiddify (127.0.0.1 : 12334). Then in the exceptions list, add the domains that you don't want a proxy for. 

![image](https://github.com/user-attachments/assets/dd6d5de7-0784-414d-a464-7a0d4d94cada)

![image](https://github.com/user-attachments/assets/6273628d-ef1f-401f-b2f9-ea3804912b83)
